orchestra/bots/staffbot.py

```diff
@@ -63,25 +76,83 @@ def validate(self, data):
                             fieldname, value))
         return data
 
+    def format_slack_message(self, text,
+                             attachments=None,
+                             response_type='ephemeral'):
+        """
```

Comment:
better to use this docstring format (no tab beyond triple quotes, and either backticks on variable names or put them in an attribute list:

https://github.com/unlimitedlabs/orchestra/blob/master/orchestra/models/core/models.py#L54